extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php

```diff
@@ -1,81 +1,83 @@
-<nav class="global-navigation" id="globalNavigation">
-	<div class="global-navigation-container">
-		<div class="wikia-logo-container table-cell">
-			<a href="<?= htmlspecialchars( $centralUrl ) ?>"
-				class="wikia-logo final-url"
-				rel="nofollow"
-				data-id="wikia-logo">
-				<img src="<?= $wg->BlankImgUrl ?>"
-					height="24"
-					width="91"
-					alt="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>"
-					title="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>">
-				<? if ( $isFandomExposed ): ?>
-					<span class="wikia-logo__subtitle"><?= wfMessage( 'global-navigation-home-of-fandom' )->escaped() ?></span>
-				<? endif; ?>
-			</a >
-		</div >
-		<?php if ( !empty( $menuContents['hubs'] ) && is_array( $menuContents['hubs'] ) ): ?>
-			<div class="table-cell hubs-links" data-visibility="desktop">
-				<?php foreach ( $menuContents['hubs'] as $hub ): ?>
-					<a class="cell-link top-level" href="<?= $hub['href']; ?>"
-					   data-tracking-label="hub-<?= $hub['specialAttr']; ?>">
-						<?= $hub['textEscaped']; ?>
+<div class="global-navigation-wrapper">
+	<nav id="globalNavigation" class="global-navigation">
+		<div class="global-navigation-container">
+			<div class="wikia-logo-container table-cell">
+				<a href="<?= htmlspecialchars( $centralUrl ) ?>"
+					class="wikia-logo final-url"
+					rel="nofollow"
+					data-id="wikia-logo">
+					<img src="<?= $wg->BlankImgUrl ?>"
+						height="24"
+						width="91"
+						alt="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>"
+						title="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>">
+					<? if ( $isFandomExposed ): ?>
+						<span class="wikia-logo__subtitle"><?= wfMessage( 'global-navigation-home-of-fandom' )->escaped() ?></span>
+					<? endif; ?>
+				</a >
+			</div >
+			<?php if ( !empty( $menuContents['hubs'] ) && is_array( $menuContents['hubs'] ) ): ?>
+				<div class="table-cell hubs-links" data-visibility="desktop">
+					<?php foreach ( $menuContents['hubs'] as $hub ): ?>
+						<a class="cell-link top-level" href="<?= $hub['href']; ?>"
+						   data-tracking-label="hub-<?= $hub['specialAttr']; ?>">
+							<?= $hub['textEscaped']; ?>
+						</a>
+					<?php endforeach; ?>
+				</div>
+			<?php endif; ?>
+			<div class="table-cell">
+				<div class="explore-wikia-entry-point" id="exploreWikiaEntryPoint">
+					<a class="cell-link top-level final-url"
+						href="<?= $isFandomExposed ? $menuContents['exploreWikia']['href'] : '#' ?>"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "exploreWikia" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)